LoggingKata/TacoParser.cs

```diff
@@ -9,10 +11,79 @@ public class TacoParser
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+            //logger.LogInfo("Begin parsing");
+
+            if(line == null)
+            {
+                logger.LogWarning("Line was null");
+                return null;
+            }
+            // Take your line and use line.Split(',') to split it up into an array of strings, separated by the char ','
+            var cells = line.Split(',');
 
             // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            // If your array.Length is less than 3, something went wrong
+            if (cells.Length != 3)
+            {
+                // Log that and return null
+                logger.LogWarning("The length was the array less than 3");
```

Comment:
The log will send a warning of "The length of the array was less than 3" even if the length is greater than 3